web/src/postprocess_content.ts

```diff
@@ -55,23 +55,20 @@ export function postprocess_content(html: string): string {
         if (elt.querySelector("img") || elt.querySelector("video")) {
```

Comment:
This section feels like it it might benefit from local variables, rather than calling `elt.querySelector("img")` like 5 times. Not a new issue, though.